superglue/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "outDir": "./built",
```

Comment:
I would like to continue using esbuild for the build process (for now as a microstep) and use typescript with noEmit. Could we remove line 4 for now?